framework/set/provisioning/nodedriver/rke1/setConfig.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/rancher/tfp-automation/framework/set/defaults"
 	aws "github.com/rancher/tfp-automation/framework/set/provisioning/providers/aws"
 	azure "github.com/rancher/tfp-automation/framework/set/provisioning/providers/azure"
+	"github.com/rancher/tfp-automation/framework/set/provisioning/providers/harvester"
```

Comment:
To be in more compliance with the other providers, can you put `harvester` in the front?